notebooks/demo3/demo3-create-tables.ipynb

```diff
@@ -0,0 +1,546 @@
+{
```

Comment:
<p>This will only download the hard coded file. We would need to add a table for all csvs in the infer folder. So we could download all the csvs here, concatenate them in one big dataframe and upload that on Trino. That big table can then be used and filtered in the Superset dashboard. WDYT?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/103/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I think we misunderstood the scope of https://github.com/os-climate/aicoe-osc-demo/issues/94, and were just generalizing demo1 to make sure it can interface well with the type of files outputted in demo2. But this makes sense, will add a more clear acceptance criteria in #94 and work with the entire set of csvs here</p>

Reply:
<p>@Shreyanand We saw that this prefix <code>infer</code> on S3 has files with different structures  eg: </p><p> </p><ol><li>sustainability-report-2019_predictions_relevant , 4137*.csv have the same columns</li><li>sustainability-report-2019_predictions_kpi.csv, sustainability-report-2019_predictions_kpi-checkpoint.csv have the same columns</li></ol><p> </p><p>Should the files with the 2nd structure not be in this prefix?</p>

Reply:
yes that's right, this happens when checkpoint files are also read by the inference code. We don't need to think about the structure of the checkpoint files. If we need to experiment with multiple files we can look at the results from the latest pipeline run stored here `/ocp-odh-os-demo-s3/corpdata/ESG/pipeline_run/samples_4/infer_relevance`

Reply:
sounds good, we will use this prefix then